buildenv/jenkins/testJobTemplate

```diff
@@ -193,58 +201,169 @@ ARCH_OS_LIST.each { ARCH_OS ->
 						parameters {
+							parameterSeparatorDefinition {
+							    name('GRINDER_WIKI_LINK')
+							    separatorStyle(separatorStyleCss)
+							    sectionHeader('<p><a href="https://github.com/AdoptOpenJDK/openjdk-tests/wiki/How-to-Run-a-Grinder-Build-on-Jenkins">Grinder Wiki</a></p>')
+							    sectionHeaderStyle(' border: none; margin-top: 0; margin-bottom: 0; padding-left: 5px; ')
+							}
+							parameterSeparatorDefinition {
+							    name('TEST_REPO_PARAMS')
+							    separatorStyle(separatorHelpTextStyleCss)
+							    sectionHeader('Test Repositories Parameters')
+							    sectionHeaderStyle(sectionHeaderStyleCss)
+							}
+							parameterSeparatorDefinition {
+							    name('TEST_REPO_PARAMS_HELP_TEXT')
+							    separatorStyle(separatorHelpTextStyleCss)
+							    sectionHeader('Repositories where we pull test material from. Unless you are testing test code, these do not need to be changed.')
+							    sectionHeaderStyle(sectionHeaderHelpTextStyleCss)
+							}							
 							stringParam('ADOPTOPENJDK_REPO', ADOPTOPENJDK_REPO, "AdoptOpenJDK git repo. Please use ssh for zos.")
 							stringParam('ADOPTOPENJDK_BRANCH', ADOPTOPENJDK_BRANCH, "AdoptOpenJDK branch")
 							stringParam('OPENJ9_REPO', OPENJ9_REPO, "OpenJ9 git repo. Please use ssh for zos.")
 							stringParam('OPENJ9_BRANCH', "master", "OpenJ9 branch")
 							stringParam('OPENJ9_SHA', "", "OpenJ9 sha")
-							stringParam('JDK_REPO', JDK_REPO, "JDK git repo. Please use ssh for zos.")
-							stringParam('JDK_BRANCH', JDK_BRANCH, "JDK branch")
-							stringParam('JDK_VERSION', "${JDK_VERSION}", "JDK version. i.e., 8, 11")
-							stringParam('JDK_IMPL', JDK_IMPL_BASE, "JDK Base implementation, e.g. hotspot, openj9")
+							stringParam('JDK_REPO', "", "JDK git repo. Please use ssh for zos.")
+							stringParam('JDK_BRANCH', "", "JDK branch")
+							stringParam('OPENJDK_SHA', "", "OpenJDK SHA from which to run tests")
+							stringParam('TKG_OWNER_BRANCH', "AdoptOpenJDK:master", "TestKitGen git [owner]:[branch]. Default is AdoptOpenJDK:master")
+							stringParam('TKG_SHA', "", "TKG sha")
+							stringParam('JCK_GIT_REPO', JCK_GIT_REPO, "For JCK test only")
+							booleanParam('LIGHT_WEIGHT_CHECKOUT', LIGHT_WEIGHT_CHECKOUT.toBoolean(), "Optional. Default is false for Grinders")
+							
+							parameterSeparatorDefinition {
+							    name('NON_AQA_TEST_REPOS')
+							    separatorStyle(separatorStyleCss)
+							    sectionHeader('Non-AQA Test Repositories')
+							    sectionHeaderStyle(sectionHeaderStyleCss)
+							}
+							parameterSeparatorDefinition {
+							    name('NON_AQA_TEST_REPOS_HELP_TEXT')
+							    separatorStyle(separatorHelpTextStyleCss)
+							    sectionHeader('Additional test repositories that can be overlayed onto existing AQA test material for extra testing.')
+							    sectionHeaderStyle(sectionHeaderHelpTextStyleCss)
+							}
+							stringParam('VENDOR_TEST_REPOS', "", "Optional. Addtional test repos")
```

Comment:
As as JDK_REPO and JDK_BRANCH, VENDOR_TEST_REPOS VENDOR_TEST_BRANCHES and VENDOR_TEST_DIRS have updated to be as parameters to be passed in.